docker-for-mac/apple-m1.md

```diff
@@ -43,6 +43,8 @@ The tech preview of Docker Desktop for Apple M1 currently has the following limi
 
 - The kernel may panic. If so, look in `~/Library/Containers/com.docker.docker/Data/vms/0/console.log` for a BUG or kernel panic to report.
 
+- The new Apple Virtualization framework uses port 53 (DNS) when Docker Desktop is started. This port cannot be used to bind a container port to the host.
```

Comment:
```suggestion
- The new Apple Virtualization framework uses port 53 (DNS) when Docker Desktop starts. You cannot use this port to bind a container's port to the host.
```

Reply:
Also reference https://github.com/docker/for-mac/issues/5335

Reply:
I've updated the PR with the suggested change, and also added the link to the issue.